custom_python_actions/check_compliance.py

```diff
@@ -28,9 +28,6 @@ def check_license(repo: github3.github.repo) -> bool:
         return True
     except github3.exceptions.NotFoundError:
         return False
-    except Exception as error:
```

Comment:
what happens if another exception other than `github3.exceptions.NotFoundError` is raised?

Reply:
Right, so I changed this because then I would want the CI job to fail with the error, not silently succeed. If it's an error that requires special handling, I can add that again.

Reply:
As long as we can quickly identify what caused the error based on the error returned by python, we should be  good! If not we can always add new `except` cases for us or developers to quickly understand what went wrong.
Feel free to resolve this thread if you agree!